libs/storage/src/storage.service.ts

```
@@ -0,0 +1,77 @@
+
+@Injectable()
+export class StorageService {
+  private readonly provider: IStorageProvider;
+
+  constructor() {
+    const fileStorageProvider = process.env.FILE_STORAGE_PROVIDER;
+    switch (fileStorageProvider) {
+      case 'minio':
+        this.provider = new MinioProvider();
+        break;
+      case 's3':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)